iphone/Maps/Core/EventListening/ListenerContainer.swift

```diff
@@ -3,15 +3,15 @@ import Foundation
 // TODO: define directly T as AnyObject after Swift version update
 final class ListenerContainer<T> {
   // MARK: - WeakWrapper for listeners
-  private class WeakWrapper<T> {
+  private class WeakWrapper<TT> {
```

Comment:
Maybe use not `TT`, but `H` or `U` to not confuse between T and TT?  Generic type placeholders in Swift usually don't use double letterings...

Reply:
Doesn't TT imply that it's inside the T type? :)

This is not critical at all, there is a TODO above, maybe a better fix will be to implement it and remove this internal type completely.